how-to-docs/drone-how-to.md

```diff
@@ -240,18 +240,18 @@ Push the changes to your remote repository.
 You can deploy the build you just pushed with the following command:
 
 ```bash
-$ drone deploy UKHomeOffice/<your_repo> 16 preprod
+$ drone deploy ukhomeoffice/<your_repo> 16 preprod
```

Comment:
is it not case sensitive?

Reply:
It wasn't on old drone and I don't see why that would change for new drone